generics/services/entity-management.js

```diff
@@ -129,7 +129,58 @@ const entityTypeDocuments = function (filterData = 'all', projection = 'all', us
 	})
 }
 
+// Function to find entity type documents based on the given filter, projection, and user token
+const userExtensionDocuments = function (filterData = 'all', projection = 'all', userToken) {
```

Comment:
@praveenKDass call it getUserRoleExtensionDocuments

Reply:
Changed the function name @VISHNUDAS-tunerlabs